examples/amp-list-with-form.amp.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+  <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+  <link rel="canonical" href="https://ampbyexample.com/components/amp-list/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    amp-list {
+      margin-left: 16px;
+    }
+  </style>
+</head>
+<body>
+<h1>&lt;amp-list&gt; with &lt;form&gt;</h1>
+<h2>&lt;form&gt; has referenced &lt;template&gt;s</h2>
+<template type="amp-mustache" id="successTemplate">Success! {{reply}}</template>
+<template type="amp-mustache" id="errorTemplate">Error in response!</template>
+<amp-list width="auto" height="100" layout="fixed-height" src="https://ampbyexample.com/json/examples.json" class="m1">
+  <template type="amp-mustache" id="amp-template-id">
+    <div>
+      <a href="{{url}}">{{title}}</a>
+      <form method="post" action-xhr="/form/echo-json/post" target="_blank">
```

Comment:
@danielrozenberg, @lannka was testing by using `action-xhr="{{url}}"` and noticed it was not working properly. Let's make sure template replacement actually works for whitelisted form attributes.

Reply:
Worked with @lannka to figure this out. Turns out it was a version sync issue - it's working for him now!

Reply:
awesome. Thanks.